pkgs/development/python-modules/hocr-tools/default.nix

```diff
@@ -0,0 +1,30 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lxml
+, pillow
+, reportlab
+, stdenv
+}:
+buildPythonPackage rec {
+  pname = "hocr-tools";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "tmbdev";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "14f9hkp7pr677085w8iidwd0la9cjzy3pyj3rdg9b03nz9pc0w6p";
+  };
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

Reply:
Added comment. it's due to sandboxing. The test framework uses git clone... https://github.com/tmbdev/hocr-tools/blob/master/test/tsht